src/mon/MonCommands.h

```diff
@@ -1505,3 +1505,7 @@ COMMAND_WITH_FLAG("dump_historic_slow_ops",
             "show recent slow ops",
             "mon", "r",
             FLAG(TELL))
+COMMAND_WITH_FLAG("subs",
+            "list existing subscription",
+            "mon", "r",
```

Comment:
```suggestion
            "mon", "rw",
```

This information can be sensitive. It should only be available to admins.

Reply:
done